test/router.js

```diff
@@ -1049,8 +1070,8 @@ describe('Router', function () {
 
       request(server)
         .get('/test/api/1234')
-        .expect(shouldNotHitHandle(1))
-        .expect(shouldNotHitHandle(2))
+        .expect(shouldHitHandle(1))
+        .expect(shouldHitHandle(2))
```

Comment:
Looks like this change was because something was broken it just is not covered in these tests but is covered in the express tests. Will try and port over the correct test here.